### PR TITLE
completed javascript-conditionals

### PR DESCRIPTION
**Pull request by Ortiz-Andrew**
Screenshot 2023-08-30 at 2 46 29 PM

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log('Does it start with J:', startsWithJ(string));
+
+const person = {
+  name: 'Andrew Ortiz',
+  Age: 27,
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: property names should be lowercase

File: javascript-conditionals/main.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+};
+function isOldEnoughToDrink(person) {
+  if (person >= 21) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: You should be checking the `age` property of the `person` object

File: javascript-conditionals/main.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    return false;
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: If they are 21 or older that means they are old enough to drink, so this should be `return true`

File: javascript-conditionals/main.js [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  } else {
+    return true;
+  }
+}
+console.log('old enough to drink:', isOldEnoughToDrink(person));
+
+function isOldEnoughToDrive(person) {
+  if (person >= 16) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: You should be checking the `age` property of the `person` object

File: javascript-conditionals/main.js [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    return false;
+  } else {
+    return true;
+  }
+}
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: If the person is 16 or older, that means they are old enough to drive, so you should be `return true`

File: javascript-conditionals/main.js [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+console.log('is old enough to drive: ', isOldEnoughToDrive(person));
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person < 21) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: You should be checking the `age` property of the `person` object

File: javascript-conditionals/main.js [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+    return true;
+  } else {
+    return false;
+  }
+}
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: Drinking & driving is illegal, so you should only be `return false`

File: javascript-conditionals/main.js [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+console.log(
+  'is person old enough to drink and drive: ',
+  isOldEnoughToDrinkAndDrive(person)
+);
+
+const pH = 12;
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH > 7) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: This should be greater than or equal to 0 & less than 7

File: javascript-conditionals/main.js [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+    return 'acid';
+  } else if (pH < 7 || pH < 14) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: This should be greater than 7 or less than or equal to 14

File: javascript-conditionals/main.js [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+    return 'base';
+  } else if (pH > -1) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: This shouldn't have any conditional. This should just be an `else`

File: javascript-conditionals/main.js [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+    case 'yakko':
+    case 'wakko':
+      return "We're the Warner brothers!";
+    case 'dot':
+      return "I'm cute~";
+    default:
+      return 'Goodnight everybody!';
+  }
+}
+console.log('introduce Warner Bros: ', introduceWarnerBro(name));
```
Comment (shawnkost) [inline javascript-conditionals/main.js:94]: You should be passing in a string when you call the function, like `'wakko'`

File: javascript-conditionals/main.js [hunk 12]
```diff
@@ -0,0 +1,126 @@
...
+    case 'horror':
+      return 'The Exorcist';
+    case 'drama':
+      return 'Whiplash';
+    case 'musical':
+      return 'La La Land';
+    case 'scifi':
+      return 'Dune';
+    case 'random':
+      return 'Genre not recognized. Choose between action, comedy, horror, drama, musical, or scifi';
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: This should actually be your `default` clause, and you shouldn't have a `random` clause

File: javascript-conditionals/main.js [hunk 13]
```diff
@@ -0,0 +1,126 @@
...
+    default:
+      return 'So many Movies! ';
+  }
+}
+console.log('recommened movies:', recommendMovie(genre));
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: You should be passing in a string for whatever genre you want, not passing in a whole object

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here Andrew!

**Comment by Ortiz-Andrew** [comment]
Screenshot 2023-08-30 at 8 12 01 PM

File: javascript-conditionals/main.js [hunk 14]
```diff
@@ -35,31 +35,31 @@ console.log('Does it start with J:', startsWithJ(string));
...
 console.log('is old enough to drive: ', isOldEnoughToDrive(person));
 
 function isOldEnoughToDrinkAndDrive(person) {
-  if (person < 21) {
-    return true;
-  } else {
+  if (person.age >= 16 && person.age >= 21) {
     return false;
+  } else {
+    return true;
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: This should be `return false`. We only want to `return false` from this function

File: javascript-conditionals/main.js [hunk 15]
```diff
@@ -72,11 +72,11 @@ const pH = 12;
 function categorizeAcidity(pH) {
   if (pH === 7) {
     return 'neutral';
-  } else if (pH > 7) {
+  } else if (pH > 7 && pH >= 0) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: This should be less than 7

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just a couple more tweaks!

**Comment by Ortiz-Andrew** [comment]
fixed

File: javascript-conditionals/main.js [hunk 16]
```diff
@@ -74,7 +72,7 @@ function categorizeAcidity(pH) {
     return 'neutral';
   } else if (pH > 7 && pH >= 0) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: This should be less than 7

File: javascript-conditionals/main.js [hunk 17]
```diff
@@ -74,7 +72,7 @@ function categorizeAcidity(pH) {
...
     return 'acid';
-  } else if (pH > 7 && pH <= 14) {
+  } else if (pH > 7 && pH < 7) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: This should still be `<= 14`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**